tools/run-wpt.mjs

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright 2023 Google LLC.
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {spawnSync} from 'child_process';
+import {cpus} from 'os';
+import {dirname} from 'path';
+
+process.chdir(dirname(dirname(process.argv[1])));
+
+function log(message) {
+  // eslint-disable-next-line no-console
+  console.log(`(${process.argv[1]}) ${message}`);
+}
+
+function usage() {
+  log(
+    `Usage: [CHROMEDRIVER=<true | false>] [HEADLESS=<true | false>] [UPDATE_EXPECTATIONS=<true | false>] ${process.argv[1]} [webdriver/tests/bidi/[...]]`
+  );
+}
+
+if (
+  process.argv.length > 2 &&
```

Comment:
It would be better to look at all args rather then only the second.

Reply:
Considering this only takes one argument right now, we can migrate to yargs for this script later.

Reply:
It's an `.includes` statement. No need for yargs